provider/bidengine/service.go

```diff
@@ -129,29 +129,26 @@ loop:
 			break loop
 
 		case ev := <-s.sub.Events():
-			switch ev := ev.(type) {
-			case mtypes.EventOrderCreated:
+			if evt, ok := ev.(mtypes.EventOrderCreated); ok {
```

Comment:
did it complain about this?

Reply:
should not be in here

Reply:
It did about switch. or you mean `evt` rename?

Reply:
yeah, the `evt` name.